packages/appstore/src/components/app.scss

```
@@ -1,3 +1,42 @@
+main {
+    @include desktop {
+        height: calc(100vh - #{$HEADER_HEIGHT} - #{$FOOTER_HEIGHT});
+    }
+    @include mobile {
+        height: calc(100vh - #{$MOBILE_HEADER_HEIGHT});
+    }
+}
+
 .dw-dashboard {
     height: 100%;
 }
+
+@keyframes triggerAnimation {
+    10% {
+        transform: rotate(10deg);
+    }
+    20% {
+        transform: rotate(-10deg);
+    }
+    30% {
+        transform: rotate(10deg);
+    }
+    40% {
+        transform: rotate(-10deg);
+    }
+    50% {
+        transform: rotate(10deg);
+    }
+    60% {
+        transform: rotate(-10deg);
+    }
+    70% {
+        transform: rotate(10deg);
+    }
+    80% {
+        transform: rotate(-10deg);
+    }
+    90% {
+        transform: rotate(10deg);
+    }
```

```suggestion
    from {
        transform: rotate(10deg);
    }
  to {
        transform: rotate(-10deg);
    }
```
Part 1: Since the value is alternating it is cleaner to do it this way. For alternating between these two values you can actually control it using the `animation` property where you use this keyframe